tutorials/packaging/create-your-first-snap/create-your-first-snap.md

```diff
@@ -148,10 +158,12 @@ Let's go through this line by line:
   - **grade**: Can be used by the publisher to indicate the quality confidence in the build. The
     store will prevent publishing 'devel' grade builds to the 'stable' channel.
 
-  - **confinement**: Can be either 'devmode' or 'strict'. A newly-developed snap should start out
+  - **confinement**: Can be either 'devmode',  'strict', or 'classic'. A newly-developed snap should start out
     in `devmode`. Security requirements can get in the way during development and 'devmode' eases
     these requirements. Security aspects like confinement can be addressed once the snap is
-    working.
+    working.  If a snap cannot be confined due to technical reasons, it may also be developed under `classic`
```

Comment:
Thanks for adding further clarification on this! I think it may be worth taking it further, so that there's no ambiguity (or perceived risk) about publishing classic snaps.  I've changed the paragraph slightly and added a note about the review process:

If there are technical issues that obstruct the confinement of a snap, it may also be developed and released using `classic` confinement. *Classic* imposes no additional restrictions and effectively grants device ownership to the snap. Consequently, snaps using *classic* confinement require a manual review before being released to the store (see [Classic confinement review process](https://forum.snapcraft.io/t/process-for-reviewing-classic-confinement-snaps/1460) for further details).

However, in this tutorial we will only focus on `devmode` and `strict` confinement.